.github/workflows/optaplanner-pr.yml

```
@@ -0,0 +1,57 @@
+# Tests RHBOP productized profile builds on PRs
+name: Optaplanner
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
+    branches:
+      - main
+      - 8.*
+    paths-ignore:
+      - 'LICENSE*'
+      - '.gitignore'
+      - '**.md'
+      - '**.adoc'
+      - '*.txt'
```

according to PR flow https://github.com/kiegroup/drools/blob/main/.github/workflows/drools-pr.yml#L6
```suggestion
      - '*.txt'
      - 'docsimg/**'
```